apps/mull-ui/src/app/pages/event-page/event-page-info/event-page-info.tsx

```diff
@@ -105,52 +105,45 @@ export const EventPageInfo = ({
           open={modalIsOpen}
           onClose={() => setIsOpen(false)}
           classes={{
-            paperWidthSm: 'cancel-event-container',
+            paperWidthSm: 'cancel-modal-container',
           }}
           maxWidth="sm"
         >
-          <div className="cancel-modal-container">
-            <p className="cancel-event-title">Cancel event?</p>
-            <FontAwesomeIcon
-              onClick={() => setIsOpen(false)}
-              icon={faTimes}
-              className="event-page-icon color-grey close-button"
-            />
-            <br />
-            <img
-              className="event-image"
-              data-testid="event-page-image"
-              src={eventImageURL ? eventImageURL : mediaUrl(event)}
-              alt="Event Page"
-            />
-            <br />
-            {/*TODO: Cancel event functionality */}
-            <MullButton className="event-page-button event-page-cancel-button" type={buttonType}>
-              Yes
-            </MullButton>
-            <MullButton
-              onClick={() => setIsOpen(false)}
-              className="event-page-button event-page-cancel-button"
-              type={buttonType}
-            >
-              No
-            </MullButton>
-          </div>
+          <button className="close-button" onClick={() => setIsOpen(false)}>
+            <FontAwesomeIcon icon={faTimes} className="color-grey" size="2x" />
+          </button>
+
+          <p className="cancel-event-title">Cancel event?</p>
+
+          <img
+            className="cancel-modal-event-image"
+            data-testid="event-page-image"
+            src={eventImageURL ? eventImageURL : mediaUrl(event)}
+            alt="Event Page"
+          />
+
+          {/*TODO: Cancel event functionality */}
+          <MullButton className="event-page-button event-page-cancel-button" type={buttonType}>
+            Yes
+          </MullButton>
+          <MullButton
+            onClick={() => setIsOpen(false)}
+            className="event-page-button event-page-cancel-button"
+            type={buttonType}
+          >
+            No
+          </MullButton>
         </Dialog>
       }
-      {isEventOwner ? (
-        /* TODO: Make cancel event functional*/
-        <div className="info-row" onClick={() => setIsOpen(true)}>
-          <FontAwesomeIcon icon={faTrash} className="event-page-icon color-grey" />
-          <p className="row-text">Cancel Event</p>
-        </div>
-      ) : (
+      {isEventExpired ? null : (
```

Comment:
💡When reviewing an event I'm creating, if I accidentally put a past datetime, the button disappears. I agree we should validate the form and not even allow this to happen, but I think the button should always show if you're in review mode.
```suggestion
      {isEventExpired && !isReview ? null : (
```